addons-cluster/etcd/values.schema.json

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "nameOverride": {
+      "title": "Name Override",
+      "description": "Override the name of the chart",
+      "type": "string",
+      "default": ""
+    },
+    "fullnameOverride": {
+      "title": "Full Name Override",
+      "description": "Override the full name of the chart",
+      "type": "string",
+      "default": ""
+    },
```

Comment:
I don't think these name fields should be set by user, it can be set by the helm release name that is enough.